docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -309,6 +509,34 @@ sdlManager.sendRPC(buttonPress);
 ```
 !@
 
+You should wrap any setting data RPC in the handler of the `SDLGetInteriorVehicleDataConsent` request.
+
+!!! Note
+`SDLGetInteriorVehicleDataConsent` is for Core v6.0+ only.
+!!! 
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager sendRequest:[[SDLGetInteriorVehicleDataConsent alloc] initWithModuleType:@"<#ModuleType#>" moduleIds:@[@"<#ModuleID#>"]] withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    SDLGetInteriorVehicleDataConsentResponse *res = (SDLGetInteriorVehicleDataConsentResponse *)response;
+    if (!res.allowed[0].boolValue) { return; }
+    <#Set Module Data#>
+}];
+```
+
+##### Swift
+```swift
+sdlManager.send(request: SDLGetInteriorVehicleDataConsent(moduleType: "<#ModuleID#>", moduleIds: ["<#ModuleID#>"]), responseHandler: { (request, response, error) in
+    guard let res = response as? SDLGetInteriorVehicleDataConsentResponse else { return }
+    let allowed = res.allowed as! [NSNumber]
+    if(allowed[0].boolValue) { return }
+    <#Set Module Data#>
+})
+```
+!@
+
 ### Subscribing to Changes
```

Comment:
It's weird that this is down here instead of with `Getting Module Data` and it basically just duplicates information from there. Instead of `Getting Module Data`, you can change it to `Subscribing to Module Data` and add small sub-section in there on how to get one-time data.